lib/auth-middleware/auth_middleware/__init__.py

```diff
@@ -23,6 +23,7 @@
 import requests
 from connexion.exceptions import OAuthProblem
 from flask import request, g
+from requests_oauthlib import OAuth2Session
```

Comment:
requests_oauthlib needs to be added to Auth Lib requirements.txt

Reply:
totally forgot about it. Thanks for catching.